repos/system_upgrade/el7toel8/actors/checkntp/actor.py

```diff
@@ -0,0 +1,74 @@
+from leapp.actors import Actor
+from leapp.libraries.stdlib import call
+from leapp.models import CheckResult, InstalledRedHatSignedRPM, NtpMigrationDecision
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+import os
+from subprocess import CalledProcessError
+
+
+class CheckNtp(Actor):
+    name = 'check_ntp'
+    description = 'Check if ntp and/or ntpdate configuration needs to be migrated.'
+    consumes = (InstalledRedHatSignedRPM,)
+    produces = (CheckResult, NtpMigrationDecision)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def check_service(self, name):
+        active = False
+        enabled = False
+        try:
+            ret = call(['systemctl', 'is-active', name])
+            active = True
+            self.log.debug('{} is active'.format(name))
+        except CalledProcessError:
+            self.log.debug('{} is not active'.format(name))
+
+        try:
+            ret = call(['systemctl', 'is-enabled', name])
```

Comment:
Can 2 check procedures be merged into one? Or reuse code?

Reply:
This was simplified in the latest commit.